src/Jank/Scorepads/ScorepadList.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useReducer } from 'react';
+import useScorepads from '../../hooks/useScorepads';
+import ScorepadItem from './Scorepad';
+import { setScorepads } from './actions';
+import reducer from './reducer';
+import { ErrorText } from './ui';
+
+const ScorepadList = () => {
+  const [error, apiScorepads] = useScorepads('JanK');
+  const [scorepads, dispatch] = useReducer(reducer, []);
+
+  useEffect(() => {
+    if (apiScorepads !== null) {
+      dispatch(setScorepads(apiScorepads));
+    }
+  }, [apiScorepads]);
+
+  scorepads.reverse();
+
+  if (error) {
+    return <ErrorText>Scorepads konnten leider nicht geladen werden</ErrorText>;
+  }
+  return (
+    <>
+      <h3>Bestehende Spiele:</h3>
+      <div>
+        {scorepads && scorepads.map((scorepad) => (
```

Comment:
Since the initial value in the reducer cannot be `null` or `undefined`, you don't need to check for `scorepads` here, I'd say. It should be an invariant that `scorepads` will always be set. If `scorepads` is an empty array, we can still call `scorepads.map()`, it just will be a no-op (which is fine). So this is probably a tautology, unless I'm missing something.